scripts/O365_Kill_Active_and_Oauth_Sessions/Readme.md

```diff
@@ -0,0 +1,39 @@
+## kill_sessions.ps1
+
+### Use case
+
+Kill active O365 session and Oauth session refresh tokens
+
+### Brief description 
+
+Pointer to Microsoft documentation about killing sessions
+
+### Data sensitivity
+
+None.
+
+### Local environment requirements
+
+Powershell.
+
+### License and/or permissions required
+
+None.
+
+### How to run the script
+
+This will kill the refresh tokens: Revoke-AzureADUserAllRefreshToken
```

Comment:
Possibly look at formatting the command call outs into a link pointing to the docs.microsoft.com documentation.